crates/bevy_ecs/src/event/mutator.rs

```diff
@@ -6,9 +6,10 @@ use bevy_ecs::{
     system::{Local, ResMut, SystemParam},
 };
 
-/// Mutably reads events of type `T` keeping track of which events have already been read
-/// by each system allowing multiple systems to read the same events. Ideal for chains of systems
-/// that all want to modify the same events.
+/// Tracks which events have been read by which system, allowing multiple systems to read the same
+/// events.
+///
+/// Ideal for chains of systems that all want to modify the same events.
```

Comment:
The mutable precision was lost here. Intentional?

Reply:
Mostly it was difficult to fit this all on one line; the thought process was that "modify" implies mutability here.